draft-ietf-tls-tls13.md

```diff
@@ -2939,6 +2939,14 @@ servers MUST process the client's ClientHello and then immediately
 send the ServerHello, rather than waiting for the client's
 EndOfEarlyData message.
 
+When using 0-RTT, a client MUST send its Finished message in response to
```

Comment:
@kaduk Dropping the "When using 0-RTT, " from the start might help alleviate your concern. That gets rid of that qualification. Optionally also add "always", e.g. "A client MUST always send its Finished..."